templates/microsoft_cloud_for_industry/financial_services_landing_zone/variables.tf

```diff
@@ -0,0 +1,327 @@
+  }
+}
+
+variable "policy_exemptions" {
+  type = map(object({
+    name                            = string
+    display_name                    = string
+    description                     = string
+    management_group_id             = string
+    policy_assignment_id            = string
+    policy_definition_reference_ids = optional(list(string))
+    exemption_category              = optional(string, "Mitigated")
+  }))
+  default     = {}
+  description = "(Optional) list of policy exemptions."
+}
+
+variable "express_route_gateway_config" {
+  type = object({
+    name                            = optional(string)
+    gatewayType                     = optional(string)
+    sku                             = optional(string)
+    vpnType                         = optional(string)
+    vpnGatewayGeneration            = optional(string)
+    enableBgp                       = optional(bool)
+    activeActive                    = optional(bool)
+    enableBgpRouteTranslationForNat = optional(bool)
+    enableDnsForwarding             = optional(bool)
+    asn                             = optional(number)
+    bgpPeeringAddress               = optional(string)
+    peerWeight                      = optional(number)
+    vpnClientConfiguration = optional(object({
+      vpnAddressSpace = optional(list(string))
+    }), {})
+  })
+  default     = { "name" : "noconfigEr" }
+  description = "(Optional) configuration options for the ExpressRoute Gateway."
+}
+
+variable "vpn_gateway_config" {
+  type = object({
+    name                            = optional(string)
+    gatewayType                     = optional(string)
+    sku                             = optional(string)
+    vpnType                         = optional(string)
+    vpnGatewayGeneration            = optional(string)
+    enableBgp                       = optional(bool)
+    activeActive                    = optional(bool)
+    enableBgpRouteTranslationForNat = optional(bool)
+    enableDnsForwarding             = optional(bool)
+    asn                             = optional(number)
+    bgpPeeringAddress               = optional(string)
+    peerWeight                      = optional(number)
+    vpnClientConfiguration = optional(object({
+      vpnAddressSpace = optional(list(string))
+    }), {})
+  })
+  default     = { "name" : "noconfigVpn" }
+  description = "(Optional) configuration options for the VPN Gateway."
+}
+
+variable "deploy_bastion" {
+  type        = bool
+  default     = true
+  description = "Toggles deployment of Azure Bastion. True to deploy, otherwise false. (e.g true)"
+}
+
+variable "landing_zone_management_group_children" {
+  type = map(object({
+    id          = string
+    displayName = string
+  }))
+  default     = {}
+  description = "(Optional) array of child management groups to deploy under the FSI Landing Zone's management group."
+}
+
+# tflint-ignore: terraform_unused_declarations
+variable "architecture_definition_template_path" {
```

Comment:
Given we are needing to use these tflint ignore rules, it seems that these settings should be in the bootstrap module instead?

Reply:
We will likely make this change for the next release as moving it would be a breaking change for SLZ.